tree/ntuple/v7/inc/ROOT/RColumnElement.hxx

```diff
@@ -105,6 +121,39 @@ public:
    std::size_t GetPackedSize(std::size_t nElements) const { return (nElements * GetBitsOnStorage() + 7) / 8; }
 };
 
+/**
+ * Base class for columns whose on-storage representation is little-endian.
+ * The implementation of `Pack` and `Unpack` takes care of byteswap if the memory page is big-endian.
+ */
+template <typename CppT>
+class RColumnElementLE : public RColumnElementBase {
+public:
+#ifdef R__BYTESWAP
```

Comment:
I would propose we rename this to `R__LITTLE_ENDIAN`, and define `R__BYTESWAP` for (`TTree`) backward compatibility. But it's confusing to see `R__BYTESWAP` being the base without byteswap (yes I know why, but it;'s still confusing).

Reply:
I mostly agree :slightly_smiling_face: - see the discussion above and the previous revision of the code.  Can we do that on a follow-up PR, as that is not only related to RNTuple?